test/ui/cypress/e2e/deployment_driver.cy.js

```diff
@@ -1,15 +1,8 @@
-import { verifyRequiredEnvVariables } from './environment_helpers'
+/// <reference types="cypress" />
 
 describe('Deployment driver web UI', () => {
-  let deploymentDriverUrl, username, password
-
-  before(() => {
-    ({deploymentDriverUrl, username, password} = verifyRequiredEnvVariables())
-  })
-
   beforeEach(() => {
-    cy.viewport(1920,1080)
```

Comment:
This is important to get reasonable viewport setting. Let's put it back.

Reply:
@tznamena , how about that I configure them in the `cypress.config.js` file?

Reply:
Keeping it in the test spec file keeps it in one place, we don't want the next developer wondering where and how it is set. But either one works.

Reply:
Sounds good. I will keep it in the test spec file.